bindings/java/iota-client-java/examples/ExampleProject/src/WrongSeedConversionSecretManager.java

```diff
@@ -0,0 +1,104 @@
+import com.google.gson.JsonObject;
+import org.iota.Client;
+import org.iota.apis.NodeIndexerApi;
+import org.iota.types.*;
+import org.iota.types.ids.BlockId;
+import org.iota.types.ids.OutputId;
+import org.iota.types.secret.GenerateAddressesOptions;
+import org.iota.types.secret.GenerateBlockOptions;
+import org.iota.types.secret.SeedSecretManager;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/*
+An incorrect seed conversion from Java to Rust in February 2022 resulted in incorrectly derived addresses. See https://github.com/iotaledger/iota.rs/pull/800 for more details.
+This example shows how to access and migrate the funds located on the incorrectly derived addresses.
+This example will try to migrate funds from the first 50 addresses of the seed.
+ */
+public class WrongSeedConversionSecretManager {
+
+    private static final String DEFAULT_TESTNET_NODE_URL = "http://localhost:14265";
+    private static ClientConfig config = new ClientConfig("{ \"nodes\": [\"" + DEFAULT_TESTNET_NODE_URL + "\" ], \"nodeSyncEnabled\": false}");
+
+    public static void main(String[] args) throws ClientException {
+        testAddressDerivation();
+        migrate();
+    }
+
+    public static void testAddressDerivation() throws ClientException {
```

Comment:
Is this an actual test ? I don't know Java but this looks like a regular class method ?

Reply:
good point.. it should actually be a test.. I think we should just move it to the actual library tests